io-engine/src/bin/io-engine-client/bdev_cli.rs

```diff
@@ -232,15 +238,21 @@ async fn share(mut ctx: Context, args: &ArgMatches<'_>) -> crate::Result<()> {
     let allowed_hosts =
         args.values_of_lossy("allowed-host").unwrap_or_default();
 
-    let response = ctx
-        .bdev
-        .share(BdevShareRequest {
-            name,
-            proto: protocol,
-            allowed_hosts,
-        })
-        .await
-        .context(GrpcStatus)?;
+    // let response = ctx
```

Comment:
Shall we consider removing these commented lines ?

Reply:
I am yet to test all changes . Hence kept older calls. WIll test it tomorrow and remove.

Reply:
Removed commented lines